packages/core/admin/admin/src/index.ts

```diff
@@ -76,3 +78,5 @@ export type { CMAdminConfiguration } from './types/adminConfiguration';
  */
 
 export { Layouts, type LayoutProps } from './components/Layouts/Layout';
+
+export type PluginDefinition = StrapiAppPlugin;
```

Comment:
Love this change, maybe we can update the plugin boilerplates to use it? Both for TS and JS with the jsdoc just like you did

Reply:
Yes next step is add a `definePlugin` function to make it even better but I want to separate some of the concerns :)